yarn.lock

```diff
@@ -13890,7 +13963,12 @@ write-file-atomic@^4.0.0, write-file-atomic@^4.0.1, write-file-atomic@^4.0.2:
     imurmurhash "^0.1.4"
     signal-exit "^3.0.7"
 
-ws@8.13.0, ws@8.17.0, ws@^7.0.0, ws@^8.11.0, ws@^8.13.0, ws@^8.17.1:
+ws@^7.0.0:
```

Comment:
this version that has a CVE vulnerability was re-introduced when you removed the ws from the resolutions? Why remove it?

Reply:
The lockfile sets the version `7.5.10` for the range `^7.0.0`. The version doesn't have a CVE vulnerability: https://github.com/zendesk/copenhagen_theme/security/dependabot/76. I also checked with `yarn audit` locally, and no audit is reported for the `ws` package.

Setting a fixed value in `resolution` could cause issues in the future. For example, the fixed version we choose could have some new vulnerabilities, or an updated dependency will require a new major version and we would force it to use the v8.x. I think it is better not to use it unless there is no other way.

Reply:
oh I see, it doesn't matter that this reads ws@^7.0.0, because the resolution is 7.5.10. Thanks for explaining 👍